testing/screenshots/make-screenshot.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Usage:
+ * ```
+ * phantomjs --ssl-protocol=any --ignore-ssl-errors=true --load-images=true \
+ *     make-screenshot.js <BASE> <PATH> <RESULT FILE> <DEVICE>
+ * ```
+ * E.g.
+ * ```
+ * phantomjs --ssl-protocol=any --ignore-ssl-errors=true --load-images=true \
+ *     make-screenshot.js "http://localhost:8000" \
+ *     "/examples.build/everything.amp.max.html" \
+ *     "everything.png" "iPhone6+"
+ * ```
+ */
+
+/* TODO(dvoytenko): open via gulp
++    "gulp-image-diff": "~0.3.0",
++    "gulp-shell": "~0.4.3",
```

Comment:
lets try and use `gulp-exec` instead `gulp-shell` seems to be blacklisted by the gulp project cause its a `duplicate` (not that it has anything malicious)

https://github.com/gulpjs/plugins/blob/master/src/blackList.json#L15